packages/itwin/grouping-mapping-widget/src/components/Groups/GroupsVisualization.tsx

```diff
@@ -4,18 +4,21 @@
 *--------------------------------------------------------------------------------------------*/
 import type { Group } from "@itwin/insights-client";
 import React, { useCallback, useEffect, useMemo, useState } from "react";
+import type { OverlappedElementGroupPairs } from "../context/GroupHilitedElementsContext";
 import { useGroupHilitedElementsContext } from "../context/GroupHilitedElementsContext";
 import {
-  getHiliteIdsFromGroups,
+  generateOverlappedGroups,
+} from "./groupsHelpers";
```

Comment:
Merge the imports. There are 2 separate imports from `./groupsHelpers`.

Reply:
Merged.